src/wp-includes/media.php

```diff
@@ -5627,6 +5627,25 @@ function wp_get_loading_optimization_attributes( $tag_name, $attr, $context ) {
 
 	$loading_attrs = array();
```

Comment:
See other comments, this can be removed (or rather initialized _after_ applying the filter).

Reply:
This variable needs to be moved below the filter.